src/ServicePulse.Host/vue/src/components/failedmessages/MessageGroupList.vue

```diff
@@ -412,22 +449,38 @@ defineExpose({
-                                    <div class="progress-bar progress-bar-striped active" role="progressbar" aria-valuenow="{{group.workflow_state.total}}" aria-valuemin="0" aria-valuemax="100" :style="{ 'min-width': '2em', width: group.workflow_state.total + '%' }">{{ group.workflow_state.total }}%</div>
+                                    <div
+                                      class="progress-bar progress-bar-striped active"
+                                      role="progressbar"
+                                      aria-valuenow="{{group.workflow_state.total}}"
+                                      aria-valuemin="0"
+                                      aria-valuemax="100"
+                                      :style="{ 'min-width': '2em', width: group.workflow_state.total + '%' }"
+                                    >
+                                      {{ group.workflow_state.total }}%
+                                    </div>
                                   </div>
                                 </div>
                               </div>
                             </li>
                             <li v-if="group.workflow_state.status === 'completed'">
                               <div class="retry-completed bulk-retry-progress-status">Retry request completed</div>
                               <button type="button" class="btn btn-default btn-primary btn-xs btn-retry-dismiss" v-if="group.need_user_acknowledgement == true" @click="acknowledgeGroup(group)">Dismiss</button>
-                              <div class="danger sc-restart-warning" v-if="group.workflow_state.failed"><i aria-hidden="true" class="fa fa-exclamation-triangle danger"></i> <strong>WARNING: </strong>Not all messages will be retried because ServiceControl had to restart. You need to request retrying the remaining messages.</div>
+                              <div class="danger sc-restart-warning" v-if="group.workflow_state.failed">
+                                <i aria-hidden="true" class="fa fa-exclamation-triangle danger"></i> <strong>WARNING: </strong>Not all messages will be retried because ServiceControl had to restart. You need to request retrying the remaining
+                                messages.
+                              </div>
                             </li>
                           </ul>
 
                           <div class="op-metadata">
-                            <span class="metadata"><i aria-hidden="true" class="fa fa-envelope"></i> {{ group.workflow_state.status === "completed" ? "Messages sent:" : "Messages to send:" }} {{ group.operation_remaining_count || group.count }}</span>
+                            <span class="metadata"
+                              ><i aria-hidden="true" class="fa fa-envelope"></i> {{ group.workflow_state.status === "completed" ? "Messages sent:" : "Messages to send:" }} {{ group.operation_remaining_count || group.count }}</span
```

Comment:
These (closing bracket on newline) look weird. Is this due to a significant whitespace thing inside of spans?

Reply:
appears to be prettier freaking out in handling this particular combination. Fixed by forcing our own newlines